docs/docsite/rst/conf.py

```diff
@@ -12,34 +12,26 @@
 #
 # All configuration values have a default value; values that are commented out
 # serve to show the default value.
-"""Documentation Configuration."""
 
-import sys
-import os
+import datetime
```

Comment:
Refactoring this module is out of scope. The change should only contain the integration of the new theme.